### PR TITLE
Display profile image to each page

### PR DESCRIPTION
**Pull request by kyoshidajp**
To increase visibility, I think it would be better to display image next to user name.

# Screenshot

 Queries list, for example.

## before
before_670320

## after
after_670320

# List of all changed page

- queries list
- queries search result list
- query detail
- query snippets list
- query snippet edit

# Other

Add `image_url` field to `User` model. If user logged in via Google Authentication, then picture url got as a callback will save as image_url and return as `gravatar_url`. Otherwise, Gravatar url will return as `gravatar_url`.

BTW a classname `profile__image_thumb` may not be appropriate.

### CODE REVIEW

**Comment by kyoshidajp** [comment]
Sorry, I am going to fix the test errors later.

**Comment by kocsmy** [comment]
This is a cool idea, thanks @kyoshidajp

File: redash/models.py [hunk 1]
```diff
@@ -436,6 +437,9 @@ def is_api_user(self):
 
     @property
     def gravatar_url(self):
```
Comment (arikfr) [inline redash/models.py]: Let's rename this to `profile_image_url` (here and everywhere else we use it) to be consistent.

**Comment by kyoshidajp** [comment]
@arikfr Thanks. I already changed. However, codeclimate doesn't finish. I'm not sure the reason.

**Comment by arikfr** [comment]
And it's in. Thanks!

**Pull request merged**